Entrega Proyecto 1

ChefeIdon opened the pull request:
### Entrega Proyecto 1

- Tema Elegido:  1
- Link a GitHub Pages: https://chefeidon.github.io/proyecto-1/

Las aclaraciones están en el readme.md

ChefeIdon committed:
Empezando la aplicacion online

- Se implemento la funcionalidad convertir un codigod e colores en un valor en ohms. Es un comienzo, el diseo de la página no esta terminado.
- Se agregaron librerias de Bootstrap.
- Se modifico el archivo de diseño app.css
- Se modifico el archivo de funcionalidad app.js, agregando las operaciones necesarias para poder seleccionar los colores desados y calcular el valor correspondiente. Falta implementar la parte de tolerancia de la resistencia.

ChefeIdon committed:
Mr. Omegoo - Diseño y conversión Color-Valor funcionando

- Se definió el diseño de la aplicación web, y se le aplicó a la parte de conversión de código de colores a valor.
- Se agregaron las páginas correspondientes para la conversión de código de colores a valor de resistencias con 5 y 6 bandas (aún no funcionando).
- Conversion de código de colores a valor finalizada.
- Se agrego un nombre y un logo a la aplicación web.

ChefeIdon committed:
Calculadora de color a valor terminada

-La aplicación permite elegir una configuración de colores de resistencias de 4, 5 y 6 bandas y calcular su valor de resistencia en Ohms.
-Estaba en pleno proceso de la funcionalidad inversa pero me acaban de decir que el proyecto se entregaba hoy 14/05 incusive, cuando (por error mio) yo contaba con el dia de mañana para terminarla y dejarla lista. Pero bueno, si puedo re-entregarlo completo creo que seria lo mejor.

ChefeIdon committed:
Actualización de Readme

ChefeIdon committed:
Agregado el link del video explicativo

Se agrego el link al video explicativo en el archivo readme.md

MarianoTucat reviewed (changes requested):
# Proyecto 1 - Agra

1. HTML: Mal
2. CSS: Bien
3. Javascript: Mal
4. Manejo DOM: Regular
5. Visualmente Atractiva: Muy Bien
6. Local Storage: Ausente
7. PR a tiempo: Si
8. GitHub Pages: Si
9. Video: Bien
Comentarios: Falta uso de Local Storage para almacenar los últimos 5 usos, los dos estilos diferentes. El código HTML está repetido innecesariamente, y el código JS abusa de la repetición y utiliza el atributo Title, habiendo mejores opciones. Mucho de lo  que está ahi se puede simplificar con arreglos y uso de ids para no repetir código. Lo mismo con el HTML, en vez de repetir el index 3 veces, se puede lograr lo mismo con el manejo del DOM en un único documento.
Resultado: Re-entrega

ChefeIdon commented:
Hola mariano, recibido! De paso te queria cuando es la fecha limite de
re-entrega. Saludos.

El mar., 19 may. 2020 15:39, Mariano Tucat <notifications@github.com>
escribió:

> *@MarianoTucat* requested changes on this pull request.
> Proyecto 1 - Agra
>
>    1. HTML: Mal
>    2. CSS: Bien
>    3. Javascript: Mal
>    4. Manejo DOM: Regular
>    5. Visualmente Atractiva: Muy Bien
>    6. Local Storage: Ausente
>    7. PR a tiempo: Si
>    8. GitHub Pages: Si
>    9. Video: Bien
>    Comentarios: Falta uso de Local Storage para almacenar los últimos 5
>    usos, los dos estilos diferentes. El código HTML está repetido
>    innecesariamente, y el código JS abusa de la repetición y utiliza el
>    atributo Title, habiendo mejores opciones. Mucho de lo que está ahi se
>    puede simplificar con arreglos y uso de ids para no repetir código. Lo
>    mismo con el HTML, en vez de repetir el index 3 veces, se puede lograr lo
>    mismo con el manejo del DOM en un único documento.
>    Resultado: Re-entrega
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ChefeIdon/proyecto-1/pull/1#pullrequestreview-414722924>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACXPKLYSKR4SWABFPEAAJODRSLG6XANCNFSM4NBG6SJA>
> .
>

ChefeIdon committed:
Página Mr.Omegoo terminada

- Se refactorizó la totalidad del código.
- Se simplifico, generalizó, ordenó y comentó el código correspondiente al archivo de funcionalidad js.
- Se agregaron ED (mapeos) para mantener la información de rutas de imagenes y valores de las mismas.
- Se unió en un solo archivo html la funcionalidad de convertir un código de color a un valor para los tres tipos de resistencias posibles.
- Se añadio el archivo html correspondiente al la funcionalidad inversa (a partir de un valor ingresado se determina el código de colores correspondiente).
- Se modificó el código del archivo .css
- Se agregó un index de bienvenida.
- Estaria terminada la re-entrega. Qusiera cambar el estilo de las alertas de datos ingresados inválidos. Si, lo puedo hacer lo subo.

ChefeIdon committed:
Merge branch 'master' of https://github.com/ChefeIdon/proyecto-1

ChefeIdon committed:
Ahora si... Mr.Omegoo Finalizada

- Se mejoró el estilo de la información de alerta al ingresar datos invalidos.
- Se corrigió el tamaño de un par de imágenes.

ChefeIdon committed:
Correcciones finales

- Se resolvieron algunos conflictos de resolución que presentaba la funcionalidad de obtener el código de colores a partir de un valor.
- Se modificó el formato de salida del valor ingresado.

ChefeIdon committed:
Actualización Readme.md

- Se modificó el archivo readme.md agregando las nuevas funcionalidades y el link al nuevo video explicativo (Re-entrega).

ChefeIdon commented:
### Re-entrega Proyecto 1

- Tema Elegido:  1
- Link a GitHub Pages: https://chefeidon.github.io/proyecto-1/

Las aclaraciones están en el readme.md

ChefeIdon commented:
### Re-entrega Proyecto 1

- Tema Elegido:  1
- Link a GitHub Pages: https://chefeidon.github.io/proyecto-1/

Las aclaraciones están en el readme.md

MarianoTucat commented:
Veo que graba en local storage, pero cuando cargo la página nuevamente, la historia no es recuperada. Puede ser? De ser así, te pido que lo arregles, no debería ser muy complicado.

ChefeIdon commented:
Hola Mariano, ya corregí el problema, e hice un commit con los cambios.
Necesitas que haga otro pull request?

El lun., 1 jun. 2020 a las 20:11, Mariano Tucat (<notifications@github.com>)
escribió:

> Veo que graba en local storage, pero cuando cargo la página nuevamente, la
> historia no es recuperada. Puede ser? De ser así, te pido que lo arregles,
> no debería ser muy complicado.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/ChefeIdon/proyecto-1/pull/1#issuecomment-637173298>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACXPKL6SL4JYUROXZ7XZXMLRUQYQ7ANCNFSM4NBG6SJA>
> .
>

MarianoTucat commented:
excelente, está bien así nomás

MarianoTucat commented:
# Reentrega Proyecto 1 - Agra

1. HTML: Bien
2. CSS: Bien
3. Javascript: Regular
4. Manejo DOM: Bien
5. Visualmente Atractiva: Bien
6. Local Storage: Mal
7. PR a tiempo: Si
8. GitHub Pages: Si
9. Video: Bien
Comentarios: Ok
Estrellas: 🌟
Resultado: Aprobado
